src/test/java/seedu/address/model/person/GroupTest.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+public class GroupTest {
+    @Test
+    public void constructor_null() {
+        Group group = new Group();
+        assertEquals(group.getGroupNumber(), 0);
+    }
+
+    @Test
+    public void constructor_valid() {
+        Group group = new Group(5);
+        assertEquals(group.getGroupNumber(), 5);
+    }
+
+    @Test
+    public void isValidGroup() {
+        // invalid group number
+        assertFalse(Group.isValidGroup(-1)); // negative number
+        assertTrue(Group.isValidGroup(0)); // zero
+
+        // valid group number
+        assertTrue(Group.isValidGroup(0)); // zero
+        assertTrue(Group.isValidGroup(100));
+    }
+
+    @Test
+    public void equals() {
```

Comment:
`equals` does not cover all cases in `Group::equals`

Reply:
Just to point out what could be improved, the overall coverage and important test cases are good enough.